### PR TITLE
Add BSD License Icon

### LINKED ISSUE #3541: Request: Open Source Licenses
As many simple icons are used by shields.io to incorporate into their badges, and many developers use a badge to show their open source license, it would be great to have simple icons for the most common open source licenses.

Resources for icons:
GNU license logos: https://www.gnu.org/graphics/license-logos.html
MIT Logo: https://en.wikipedia.org/wiki/MIT_License#/media/File:MIT_logo.svg
Apache Logo: https://en.wikipedia.org/wiki/Apache_License#/media/File:Apache_Software_Foundation_Logo_(2016).svg
BSD logo: https://en.wikipedia.org/wiki/BSD_licenses#/media/File:BSD_wordmark.svg
Unlicense logo: https://en.wikipedia.org/wiki/Unlicense#/media/File:PD-icon-black.svg

There are a few more linked here: https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licences

- [ ] GNU License(s)
  - [x] GFDL (#11601)
  - [ ] GPL3 or later
  - [ ] GPL3
  - [ ] LGPL
  - [ ] AGPL
- [ ] ~MIT License~ (Won't add, as logo is just that of the school. ~#7544~)
- [x] Apache License
- [x] BSD License (~#7549~ #8965 )
- [x] Unlicense (#7543)

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
bsd

**Issue:** contributes toward #3541 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from brand resources webpage https://freebsdfoundation.org/about-us/about-the-foundation/project
Color: `#AB2B28` Taken from guidelines for color palette https://freebsdfoundation.org/about-us/about-the-foundation/project

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1800,6 +1800,12 @@
             "hex": "CC092F",
             "source": "https://en.wikipedia.org/wiki/Broadcom_Inc"
         },
+        {
+            "title": "BSD",
+            "hex": "AB2B28",
+            "source": "https://freebsdfoundation.org/about-us/about-the-foundation/project",
+            "guidelines": "https://freebsdfoundation.org/about-us/about-the-foundation/project"
```
Comment (adamrusted) [inline _data/simple-icons.json:1807]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:BSD_wordmark.svg"
```
I'm not 100% about using `FreeBSD` as our source, given it's one of a few projects that originate the BSD license. For that sake, we're probably safer using the Wikimedia source.

File: icons/bsd.svg
Comment (adamrusted) [inline icons/bsd.svg:1]: When comparing to the Wikimedia source, there are [visible differences](https://github.com/simple-icons/simple-icons/assets/1830867/64c16bdc-712b-4ded-9350-197812420fd4). Could you have another go please @PratyushJayachandran 🙏

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:1807]: The wikimedia icon is from the same source
https://commons.wikimedia.org/wiki/File:BSD_wordmark.svg
image

Comment (adamrusted) [reply _data/simple-icons.json:1807]: Ignore me then!

File: icons/bsd.svg
Comment (PratyushJayachandran) [reply icons/bsd.svg:1]: Not changing since Wikimedia was also supposedly taken from the source I mentioned. Just commenting for reference.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PratyushJayachandran 🎉

**Pull request merged**